core/src/main/scala/codesearch/core/Main.scala

```diff
@@ -34,13 +34,7 @@ object Main extends IOApp {
         params <- CLI.params(args)
         config <- Config.load[IO]
 
-        dbConfig = config.db
-
-        val db = Database.forURL(
-          driver = "org.postgresql.Driver",
-          url =
-            s"jdbc:postgresql://${dbConfig.host}:${dbConfig.port}/${dbConfig.name}?user=${dbConfig.user}&password=${dbConfig.password}"
-        )
+        val db = Database.forConfig("db")
```

Comment:
First, the IDE gives you a hint. Please do not ignore it:
![image](https://user-images.githubusercontent.com/5605372/56092637-a5897100-5ecf-11e9-8c0e-66c5be8b4f88.png)
Secondly, values are better to declare at their place of use, not higher.